### PR TITLE
bump to 0.4.0

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@ ideaVersion=IC-2019.3
 # build number ranges
 # https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html
 sinceIdeaBuild=193
-projectVersion=0.3.1-SNAPSHOT
+projectVersion=0.4.0
```
Comment (jeffmaury) [inline gradle.properties:5]: Why the SNAPSHOT disappeared ?

Comment (adietish) [reply gradle.properties:5]: want to release 0.4.0, for this sake need a stable version which will get bumped to 0.4.0-SNAPSHOT once released and tagged. Missing something?

Comment (jeffmaury) [reply gradle.properties:5]: So you want to release 0.4.0 ? Then you also need a modification in plugin.xml to document changes, no ?

Comment (adietish) [reply gradle.properties:5]: ouch, forgot, again, thx for reminding me. doing.

Comment (adietish) [reply gradle.properties:5]: @jeffmaury plz re-review

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**